apps/masterbots.ai/components/routes/browse/browse-thread-blog.tsx

```
@@ -0,0 +1,221 @@
+    return sections.map((section, index): JSX.Element => {
+      // Handle numbered lists (both main and nested)
+      const numberMatch = section.match(/^(\d+(?:\.\d+)?)\.\s+(.+)/s);
+      
+      if (numberMatch) {
+        const [, number, text] = numberMatch;
+        const [title, ...contentParts] = text.split(':');
+        const isNested = number.includes('.');
+        
+        return (
+          <div 
+            key={`numbered-${index}`} 
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)